Source/WTF/wtf/text/Base64.h

```diff
@@ -42,7 +42,7 @@ enum class Base64EncodeMode : bool { Default, URL };
 //   enforce a correct number of trailing equal signs in the input.
 // - ::DefaultValidatePaddingAndIgnoreWhitespace ignores ASCII whitespace in
 //   the input. It matches <https://infra.spec.whatwg.org/#forgiving-base64>.
-enum class Base64DecodeMode { DefaultIgnorePadding, DefaultValidatePadding, DefaultValidatePaddingAndIgnoreWhitespace, URL };
+enum class Base64DecodeMode { DefaultIgnorePadding, DefaultValidatePadding, DefaultValidatePaddingAndIgnoreWhitespace, DefaultIgnoreWhitespace, URL };
```

Comment:
You added a new enum type, but did not ad a comment (like the other cases) to explain what it is for.

Reply:
Should this new enum have a name that indicates that it is a Quirk to address malformed content sent to Safari UA? I guess the name is functional in explaining what it does, so perhaps adding the context that we are working around a site bug is irrelevant.

Reply:
I applied the suggestions in this latest iteration.